pkg/semtech-loramac/contrib/semtech_loramac.c

```diff
@@ -67,32 +64,22 @@
 #endif /* USE_SEMTECH_DEFAULT_CHANNEL_LINEUP */
 #endif /* REGION_EU868 */
 
-#if defined(REGION_CN779) || defined(REGION_EU868) || \
-    defined(REGION_IN865) || defined(REGION_KR920)
-#define LORAWAN_APP_DATA_SIZE                       (16U)
-#elif defined(REGION_AS923) || defined(REGION_AU915) || \
-      defined(REGION_US915) || defined(REGION_US915_HYBRID)
-#define LORAWAN_APP_DATA_SIZE                       (11U)
-#else
-#error "Please define a region in the compiler options."
-#endif
-
-static char stack[LORAMAC_STACKSIZE];
+#define SEMTECH_LORAMAC_MSG_QUEUE                   (16U)
+#define SEMTECH_LORAMAC_LORAMAC_STACKSIZE           (THREAD_STACKSIZE_DEFAULT)
+static char _semtech_loramac_stack[SEMTECH_LORAMAC_LORAMAC_STACKSIZE];
 kernel_pid_t _semtech_loramac_pid;
 kernel_pid_t _semtech_loramac_handler_pid;
 
-RadioEvents_t radio_events;
-uint8_t dev_eui[LORAMAC_DEVEUI_LEN];
-uint8_t app_eui[LORAMAC_APPEUI_LEN];
-uint8_t app_key[LORAMAC_APPKEY_LEN];
-uint8_t nwk_skey[LORAMAC_NWKSKEY_LEN];
-uint8_t app_skey[LORAMAC_APPSKEY_LEN];
-uint8_t dev_addr[LORAMAC_DEVADDR_LEN];
+RadioEvents_t _semtech_loramac_radio_events;
```

Comment:
Sorry if I wasn't clear. exported symbols starting with underscore are against C spec. use underscore for static, remove for exported global symbols.